docs/manual/article-management/insert-tags.de.md

```diff
@@ -26,6 +26,7 @@ oder den Alias der Zielseite.
 | `{{link::login}}`         | Dieses Tag wird mit einem Link zur Anmeldeseite des aktuellen Frontend-Benutzers (falls vorhanden) ersetzt.                  |
 | `{{link_open::*}}`        | Wird mit dem öffnenden Tag eines Links zu einer internen Seite ersetzt: `{{link_open::12}}Hier klicken{{link_close}}`.       |
 | `{{link_url::*}}`         | Dieses Tag wird mit der URL einer internen Seite ersetzt: `<a href="{{link_url::12}}">Hier klicken</a>`.                     |
+| `{{link_target::*}}`      | Dieses Tag wird mit ` target="_blank" rel="noreferrer noopener"` ersetzt, wenn es sich bei der angegebenen Seite um eine externe Weiterleitungsseite handelt, und dort eingestellt ist, dass sich der Link in einem neuen Fenster öffnen soll. |
```

Comment:
The extra space right before target does not belong there, does it?

Reply:
It does. This is so that you can directly put it after the `href` attribute, for example: `<a href=""{{link_target::*}}>`

Reply:
It does belong there :). Usage:

```html
<a href="{{link_url::1}}"{{link_target::1}}>
```

Otherwise you would have an unecessary space in your markup, if the Insert Tag did not insert the space itself.

Reply:
OK, I see.Thank you. Would this be worth mentioning? something l ike 
```
Dieses Tag wird mit ` target="_blank" rel="noreferrer noopener"` ersetzt (inklusive des Leerzeichens am Anfang)
```